docs/other-components/align-score/index.md

```diff
@@ -0,0 +1,9 @@
+# What is Align Score?
+
+_"AlignScore, a metric for automatic_ __factual consistency evaluation of text pairs__" is
+introduced in [AlignScore: Evaluating Factual Consistency with a Unified Alignment Function](https://arxiv.org/abs/2305.16739)
+by Yuheng Zha, Yichi Yang, Ruichen Li and Zhiting Hu
+
+AlignScore returns a score between `0` (Not consistent)
+and `1` (Perfectly consistent) for if the given _claims_ are consistent with the
+provided _contexts_.
```

Comment:
Can we end this page with links to the child pages? It would be clearer than having to look at the page tree on the left (this probably applies to other pages too that have child pages). The absence of this is one of my main gripes with Confluence.

Reply:
Will do.

Reply:
And regarding confluence: https://confluence.atlassian.com/doc/children-display-macro-139501.html